### PR TITLE
Added instructions to setup kubescape locally

### PR DESCRIPTION
**Pull request by yrs147**
## Overview

The `CONTRIBUTING.md` file was lacking information on how to set up kubescape locally. To assist new contributors, I have added instructions for this purpose.

**Screenshots**

image

image

### CODE REVIEW

**Comment by HollowMan6** [comment]
I think we shall put a link to https://github.com/kubescape/kubescape/wiki/Building instead of duplicating the information there.

**Comment by matthyx** [comment]
There is also something wrong here, I thought we only needed `libgit` when building with the flag `gitenabled` but the documentation doesn't reflect that.
There should also be somewhere a mention that for performance reasons it's better to build with this dependency when scanning git repos.
For reference it was introduced with https://github.com/kubescape/kubescape/pull/969

**Comment by HollowMan6** [comment]
@matthyx Okay, I didn't aware that this was outdated when I moved the building instructions there through this PR: https://github.com/kubescape/kubescape/pull/1196 I will try to update the instructions accordingly then.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,6 +15,65 @@ so the maintainers are able to help guide you and let you know if you are going
...
+ OR
+  ```
+   go build -tags=static .
+   ```
+3.  Run kubescape
+```
+./kubescape
+```
+For more details you can checkout [wiki](https://github.com/kubescape/kubescape/wiki/Building)
+
```
Comment (HollowMan6) [inline CONTRIBUTING.md:76]: For this PR, we only need to attach the links to the Wiki page

```suggestion
## Build and test locally
Please follow the [instructions here](https://github.com/kubescape/kubescape/wiki/Building).
```

**Review by HollowMan6 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -91,4 +150,4 @@ More Examples:
 
 ## Fixing a commit where the DCO failed
 
-Check out [this guide](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md).
+Check out [this guide](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md).
```
Comment (HollowMan6) [inline CONTRIBUTING.md:96]: Do not remove the newline at the end of the file here.

```suggestion
Check out [this guide](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md).

```

**Review by HollowMan6 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (HollowMan6) [reply CONTRIBUTING.md:96]: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Review by HollowMan6 — APPROVED** [review]
LGTM, thanks!

**Comment by HollowMan6** [comment]
cc: @matthyx @craigbox You may want to merge this.

**Comment by HollowMan6** [comment]
Updated! https://github.com/kubescape/kubescape/wiki/Building

> @matthyx Okay, I didn't aware that this was outdated when I moved the building instructions there through this PR: #1196 I will try to update the instructions accordingly then.

**Review by craigbox — APPROVED** [review]

**Pull request merged**